README.md

```diff
@@ -17,6 +17,9 @@ The project is very much Work In Progress and will be published on maven central
 
 # Release Notes
 BOAT is still under development and subject to change.
+## 0.17.4
+* Boat Angular generator
+  * New format for Angular mocks, which are now export an array with responses.
```

Comment:
Is this change significant enough to warrant bumping to `0.18.0`?  Not sure what the versioning strategy is with BOAT...

Reply:
We discussed it with Ilya, as I understood 17 is not commonly used and is yet to come to the public, so we can introduce breaking changes there